package/src/components/AutoCompleteInput/AutoCompleteSuggestionList.tsx

```diff
@@ -31,9 +32,9 @@ export type AutoCompleteSuggestionListPropsWithContext<
 > &
   AutoCompleteSuggestionListComponentProps<StreamChatGenerics>;
 
-const SuggestionsItem: React.FC<TouchableOpacityProps> = (props) => {
-  const { children, ...touchableOpacityProps } = props;
-  return <TouchableOpacity {...touchableOpacityProps}>{children}</TouchableOpacity>;
+const SuggestionsItem: React.FC<PressableProps> = (props) => {
+  const { children, ...pressableProps } = props;
+  return <Pressable {...pressableProps}>{children}</Pressable>;
```

Comment:
its fine to change to pressable but we remove the opacity change with this 

Look into copying this component https://github.com/mrousavy/react-native-pressable-opacity/blob/master/src/PressableOpacity.tsx

mimics touchable opacity in pressable opacity..

Reply:
Done